tyrian/js/src/main/scala/tyrian/Cmd.scala

```diff
@@ -42,8 +47,12 @@ object Cmd:
       SideEffect(Sync[F].delay(thunk))
 
   /** Simply produces a message that will then be actioned. */
-  final case class Emit[Msg](msg: Msg) extends Cmd[Nothing, Msg]:
-    def map[OtherMsg](f: Msg => OtherMsg): Emit[OtherMsg] = Emit(f(msg))
+  final case class Emit[Msg](msg: Msg, afterDelay: FiniteDuration) extends Cmd[Nothing, Msg]:
```

Comment:
Sorry for being noisy, but I still think that if we can express something with a very simple combination of existing commands (`Run(IO.sleep(delay), _ => msg)`) - we should do that rather than overloading a single-purpose command.

So, I'd prefer to keep the `Emit` as is and changed `emitAfterDelay` to:

```scala
def emitAfterDelay[Msg](msg: Msg, delay: FiniteDuration): Emit[Msg] =
  Run(IO.sleep(delay), _ => msg)
```